bots/zephyr_mirror_backend.py

```diff
@@ -1004,6 +1036,7 @@ def parse_args():
     return parser.parse_args()
 
 def die_gracefully(signal, frame):
+    # type: (Any[dict[[int, type], int]], Any) -> None
```

Comment:
`frame` gets passed in by the standard library and can probably be a more specific type than `Any`.